ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.html

```diff
@@ -220,9 +220,9 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
           <br/>
         </div>
 
-        <app-search-bar [textQuery]="filterQuery" [allFieldNames]="allFieldNames" (queryText)="doFilterByQuery($event)"
-                        [showHelp]="showHelp" [filters]="sourceColumns">
-        </app-search-bar>
+<!--        <app-search-bar [textQuery]="filterQuery" [allFieldNames]="allFieldNames" (queryText)="doFilterByQuery($event)"-->
+<!--                        [showHelp]="showHelp" [filters]="sourceColumns">-->
+<!--        </app-search-bar>-->
```

Comment:
I thought we are removing that

Reply:
commenting it out for now, in case we want to add it later